nx-workspace/apps/web/src/components/atoms/core/AppBar.tsx

```diff
@@ -1 +1,4 @@
-export { AppBar, AppBarProps } from "@material-ui/core";
+import { AppBarProps as t_AppBarProps } from "@material-ui/core";
+
+export { AppBar } from "@material-ui/core";
+export type AppBarProps = t_AppBarProps;
```

Comment:
Assuming this is a warning about exporting a value with a type? Does it fix it to export them separately, e.g.:

```suggestion
import { AppBarProps as t_AppBarProps } from "@material-ui/core";

export { AppBar } from "@material-ui/core";
export type AppBarProps = t_AppBarProps;
```

```suggestion
export { AppBar } from "@material-ui/core";
export { AppBarProps } from "@material-ui/core";
```

Or, alternatively:

```suggestion
export { AppBar } from "@material-ui/core";
import { AppBarProps as MAppBarProps } from "@material-ui/core";
export type AppBarProps = MAppBarProps;
```